Ahmadreza-tahmasebi/c6.js

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+function oneSidePyramidMaker(num){
+    for(var i = 1 ; i <= num ; i++){
+        for(var j = 1 ; j <= (i * 2) - 1 ; j++){
+            process.stdout.write("*");
+        }
+        console.log();
+    }
+}
+
+function pyramidMaker(num) {
+    var middle = Math.ceil(((num * 2)-1 )/ 2)
+    var temp = [middle]
```

Comment:
No need to create a temp variable here. You could have used the middle itself as an array later.

Reply:
```
temp.push(middle + i );
```
If so, this part of the code won't work !